acpi/src/fadt.rs

```diff
@@ -128,3 +131,27 @@ impl Fadt {
         }
     }
 }
+
+/// Information about the ACPI Power Management Timer (ACPI PM Timer).
+pub struct PmTimer {
```

Comment:
(Context provided in the top-level review comment)
I think this might be better living in `platform.rs`, stored as a `Option<PmTimer` in `PlatformInfo`, and extracted from the FADT in [`PlatformInfo::new`](https://github.com/rust-osdev/acpi/blob/master/acpi/src/platform.rs#L142)